### PR TITLE
Candle led

### PR DESCRIPTION
**Pull request by moonflowerrrr**
auton LEDs have not been tested and the disabled LEDs do not work

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -123,6 +121,51 @@ public static enum ClimberPosition {
...
+            }
+
+            public int getRed() { return red; }
+            public int getGreen() { return green; }
+            public int getBlue() { return blue; }
+        }
+
+        public static enum LEDStrip {
+            CANDLE(0, 8),
+            SHOOTER(8, 0),
```
Comment (dlozon) [inline src/main/java/frc/robot/Constants.java]: We still have to finish filling out the ledstrip values

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -21,10 +21,8 @@ public static final class ControllerPorts {
         public static final int OPERATOR_PORT = 1;
     }
 
-    /** PWM ports used by blinkin LED controllers */
-    public static final class BlinkinPorts {
-        public static final int SHOOTER_LEDS_PORT = 0;
-        public static final int INTAKE_LEDS_PORT = 1;
+    public static final class LEDPorts {
+        public static final int LED_PORT = 2;
```
Comment (dlozon) [inline src/main/java/frc/robot/Constants.java]: should be named in a way that indicates that candle uses a CAN id and not a port on the rio

File: src/main/java/frc/robot/subsystems/Candle.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import com.ctre.phoenix.led.CANdle;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.LEDConstants.LEDColor;
+import frc.robot.Constants.LEDConstants.LEDStrip;
+
+public class Candle extends SubsystemBase {
+
+    public CANdle candle;
+
+    /** */
```
Comment (dlozon) [inline src/main/java/frc/robot/subsystems/Candle.java]: this class should be commented

Comment (moonflowerrrr) [reply src/main/java/frc/robot/subsystems/Candle.java]: okay g

**Pull request merged**